arches_for_science/media/js/views/components/workflows/chemical-analysis-workflow/ca-instrument-info-step.js

```diff
@@ -165,10 +169,12 @@ define([
         };
 
         params.form.reset = function(){
+            self.nameValue(null);
+            self.instrumentName(null);
```

Comment:
These needed to be cleared out because of the subscriptions that get called e.g. instrumentValue() right after this. If there is still any data here, it was preventing getting the form state to be exactly like the original empty state.

Reply:
Actually, I found a better place to do this (inside the instrumentName subscription). The first version here was too aggressively setting to null.